data/processed/hesa/total_stem_students.yaml

```diff
@@ -0,0 +1,22 @@
+api: fetch
+api_documentation: https://www.hesa.ac.uk/data-and-analysis/students
+description: Number of students enrolled full-time in STEM subjects in a NUTS2 region (definition of STEM subjects in aux folder) in the starting academic year. 
+endpoint_static: https://www.hesa.ac.uk/data-and-analysis/students/table-13.csv
+framework_group: public_rnd_capability
+is_experimental: False
+nuts_level: 2
```

Comment:
We need to add the line:
```
order: [year, nuts_id, nuts_year_spec, value.id]
```
see #131 and #130.

Reply:
Done